ethereum/spec/src/main/java/tech/pegasys/teku/spec/datastructures/blocks/versions/deneb/SignedBlockContents.java

```diff
@@ -46,7 +52,12 @@ public SignedBeaconBlock getSignedBlock() {
   }
 
   @Override
-  public Optional<List<SignedBlobSidecarOld>> getSignedBlobSidecars() {
-    return Optional.of(getField1().asList());
+  public Optional<List<KZGProof>> getKzgProofs() {
```

Comment:
I think it would be better if we have `Optional<SszList<SszKZGProof>>` and `Optional<SszList<Blob>>` similar to how we return the blobKzgCommitments in BeaconBlockBodyDeneb. It would avoid wrapping and unwrapping when using the values since most of the time we just need to loop and get the values. Same for the return type in `BlockContents`.

Reply:
Done